.travis.yml

```diff
@@ -1,24 +1,35 @@
+sudo: required
+services:
+  - docker
 language: elixir
 elixir:
-  - 1.2.5
+  - 1.3.2
 cache:
   - apt
   - directories:
     - ~/.mongodb
 before_install:
+  - rm -rf ~/.nvm
+  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.1/install.sh | bash
+  - source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
+  - npm i -g npm
```

Comment:
Yes, that was actually made after I wrote that. When I merge in master again, I will use that commit instead of my own.